references/classification/README.md

```diff
@@ -19,21 +19,19 @@ pip install pyronear
 
 ## Train
 
-### Wildfire
-
-Download Dataset from https://drive.google.com/file/d/1Y5IyBLA5xDMS1rBdVs-hsVNGQF3djaR1/view?usp=sharing
+### OpenFire
 
-This dataset is protected by a password, please contact us at contact@pyronear.org
+You can also use out opensource dataset without password
 
 ```
-python train.py WildFireLght/ --model rexnet1_0x --lr 1e-3 -b 16 --epochs 20 --opt radam --sched onecycle --device 0
+python train.py OpenFire/ --use-openfire --model rexnet1_0x --lr 1e-3 -b 16 --epochs 20 --opt radam --sched onecycle --device 0
 ```
 
-### OpenFire
+### Wildfire
 
-You can also use out opensource dataset without password
+This dataset is private, you should be part of Pyronear to access it
```

Comment:
Suggestion:
"This dataset is private for now, for now only Pyronear members can access it"